docs/resources/associated_site.md

```diff
@@ -0,0 +1,238 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_associated_site"
+sidebar_current: "docs-aci-resource-aci_associated_site"
+description: |-
+  Manages ACI Associated Site
+---
+
+# aci_associated_site #
+
+Manages ACI Associated Site
+
+  -> This resource should only be used to accomplish multi-site orchestration.
+
+
+## API Information ##
+
+* Class: [fvSiteAssociated](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvSiteAssociated/overview)
+
+* Supported in ACI versions: 3.0(1k) and later.
+
+* Distinguished Name Formats:
+  - `uni/tn-{name}/BD-{name}/stAsc`
+  - `uni/tn-{name}/ap-{name}/epg-{name}/stAsc`
+  - `uni/tn-{name}/ctx-{name}/stAsc`
+  - `uni/tn-{name}/mscGraphXlateCont/epgDefXlate-[{epgDefDn}]/stAsc`
+  - `uni/tn-{name}/out-{name}/instP-{name}/stAsc`
+
+## GUI Information ##
+
+* Location: `Not shown in UI`
+
+## Example Usage ##
+
+The configuration snippet below creates a Associated Site with only required attributes.
```

Comment:
```suggestion
The configuration snippet below creates an Associated Site with only required attributes.
```
Not sure if it is worth taking this case into account but we might want to.

Reply:
I would not do this case now, since it leads to 66 changes for which some are wrong the other way, where `an` should be `a`. In this case need to implement some vowel checking logic with overrides ( or additional logic ) in case of certain exceptions when a word sounds like a vowel. Perhaps something to note for redesign?

Reply:
Ok. Lets figure out either a new sentence or a way to take care of it in the redesign.